controller/src/beerocks/master/db/node.h

```diff
@@ -183,6 +184,28 @@ class node {
         };
         std::shared_ptr<ap_stats_params> stats_info;
         std::unordered_map<int8_t, sVapElement> vaps_info;
+
+        struct channel_scan_config {
+            bool is_enabled = false;
+            std::unordered_set<uint8_t> channel_pool; // default value: empty list
+            int interval_sec    = -1;                 //-1 (invalid)
+            int dwell_time_msec = -1;                 //-1 (invalid)
+        };
+
+        struct channel_scan_status {
+            bool scan_in_progress = false;
+            beerocks::eChannelScanErrCode last_scan_error_code =
+                beerocks::eChannelScanErrCode::CHANNEL_SCAN_SUCCESS;
+        };
+        //These members are part of the continuous scan
+        //The contiuous scan, scans every
+        channel_scan_config continuous_scan_config;
```

Comment:
Please add doxygen for each new parameter, what it is and what it is used for.

Reply:
done